src/pages/RealtimeMapPage.tsx

```diff
@@ -43,20 +42,15 @@ export const colorIcon = ({ operator_id, name }: { operator_id: string; name?: s
     className: 'my-div-icon',
     html: `
     <div class="bus-icon-container">
-      <div class="bus-icon-circle">
-        <img src="${path}" alt="${name}" />
-      </div>
-      <div class="operator-name">${name}</div>
+    <div class="bus-icon-circle">
+    <img src="${path}" alt="${name}" />
+    </div>
+    <div class="operator-name">${name}</div>
```

Comment:
please restore the indentation

Reply:
<img width="320" alt="image" src="https://github.com/hasadna/open-bus-map-search/assets/45571546/84fc3193-4f30-4db0-ab4d-a1e79efd3e3b">